.circleci/conditional_config.yml

```diff
@@ -155,6 +156,9 @@ jobs:
             cd src/api;
             bundle exec rake dev:lint:rubocop:all
       - *save_rubocop_cache
+      - run:
+          name: Test loading the App in production
+          command: RAILS_ENV=production bundle exec rake db:setup zeitwerk:check
```

Comment:
What is the point of doing `db:setup`? `zeitwerk:check` doesn't need the database to ensure classes are loaded properly.

Reply:
it does. the check some after the check for the DB...